package.json

```diff
@@ -78,9 +78,10 @@
     "setupTestFrameworkScriptFile": "./test-setup.js"
   },
   "dependencies": {
+    "clean-tag": "^1.0.4",
     "prop-types": "^15.5.10",
-    "styled-components": "^2.1.1",
-    "styled-system": "^1.0.8-0"
+    "styled-components": ">=2.1.1 || >=3.0.0",
```

Comment:
Interesting. Did not know about the `||`, and this PR inspired the curiosity to poke around on the the [**Ranges**](https://docs.npmjs.com/misc/semver#ranges) topic in the npm doc about the `||` used in dependency's version. 🙏 for this cool PR with `styled-components` update. 👍 
> A range is composed of one or more comparator sets, joined by ||. A version matches a range if and only if every comparator in at least one of the ||-separated comparator sets is satisfied by the version.